pkgs/tools/admin/pgadmin/update.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl wget jq yarn2nix yarn common-updater-scripts
+
+set -eu -o pipefail
+
+nixpkgs="$(git rev-parse --show-toplevel)"
```

Comment:
```suggestion
nixpkgs="../../../.."
```

Reply:
with this I am getting:
```
error: string '../../../..' doesn't represent an absolute path

       at «string»:1:2:

            1| (import "../../../.." { config = {}; overlays = []; }).pgadmin4.version
             |  ^
(use '--show-trace' to show detailed location information)
```

Reply:
This is basically the same error as before:
```
error: string './../../../..' doesn't represent an absolute path

       at «string»:1:2:

            1| (import "./../../../.." { config = {}; overlays = []; }).pgadmin4.version
             |  ^
(use '--show-trace' to show detailed location information)
```